scipy/optimize/_minimize.py

```diff
@@ -77,6 +78,7 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
             - 'trust-ncg'   :ref:`(see here) <optimize.minimize-trustncg>`
             - 'trust-exact' :ref:`(see here) <optimize.minimize-trustexact>`
             - 'trust-krylov' :ref:`(see here) <optimize.minimize-trustkrylov>`
+            - 'differential-evolution' :ref:`(see here) <`scipy.optimize.differential_evolution`>`
```

Comment:
Maybe here?
```suggestion
            - 'differential-evolution' :ref:`(see here) <scipy.optimize.differential_evolution>`
```

Reply:
Hmm that might not be all:
```
/home/circleci/repo/build/testenv/lib/python3.7/site-packages/scipy/optimize/_minimize.py:docstring of scipy.optimize.minimize:41: WARNING: undefined label: (see here) <`scipy.optimize.differential_evolution (if the link has no caption the label must precede a section header)
/home/circleci/repo/build/testenv/lib/python3.7/site-packages/scipy/optimize/_minimize.py:docstring of scipy.optimize.minimize:309: WARNING: undefined label: scipy.optimize.differential_evolution (if the link has no caption the label must precede a section header)
/home/circleci/repo/build/testenv/lib/python3.7/site-packages/scipy/optimize/optimize.py:docstring of scipy.optimize.show_options:51: WARNING: undefined label: scipy.optimize.differential_evolution (if the link has no caption the label must precede a section header)
```
I'll look into it but hah I'm no expert. I just had the misfortune of fighting with it a lot when working on the contributor guide.

Reply:
Unfortunately I haven't looked into method documentation references are really working. It looks like there might be other files to be added/changed; maybe following https://github.com/scipy/scipy/pull/8328/files as an example of all the changes made to add a method to `minimize` would help.